skiko/src/commonTest/kotlin/org/jetbrains/skia/ParagraphTest.kt

```diff
@@ -49,23 +46,49 @@ class ParagraphTest {
     }
 
     @Test
-    @SkipJsTarget
-    @SkipWasmTarget
     @SkipNativeTarget
-    fun layoutParagraph() = runTest {
-        singleLineMetrics("aa").let { lineMetrics -> // latin
-            assertEquals(0, lineMetrics.startIndex)
-            assertEquals(2, lineMetrics.endIndex)
-            assertEquals(2, lineMetrics.endIncludingNewline)
-            assertEquals(2, lineMetrics.endExcludingWhitespaces)
-        }
-        singleLineMetrics("яя").let { lineMetrics -> // cyrillic
-            assertEquals(0, lineMetrics.startIndex)
-            assertEquals(2, lineMetrics.endIndex)
-            assertEquals(2, lineMetrics.endIncludingNewline)
-            assertEquals(2, lineMetrics.endExcludingWhitespaces)
+    fun layoutParagraph(): TestReturnType {
+        val lineMetricsEpsilon = 0.0001f
+
+        return runTest {
```

Comment:
It's better to return runTest result right away, because of js - https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-test.html
___

I tried to change the asserts, and the test failed as expected. It's a bit surpising to me. We don't use `runTest` from coroutines lib. Maybe, that is the reason.

Reply:
Or maybe the fact it has `return` is already sufficient.

Reply:
Nevermind, the `= runTest` makes it consistently looking with the rest of the tests, so I pushed this change